pkgs/top-level/all-packages.nix

```diff
@@ -13938,6 +13938,8 @@ with pkgs;
 
   opam-installer = callPackage ../development/tools/ocaml/opam/installer.nix { };
 
+  inherit (ocamlPackages) opam2json;
```

Comment:
Why is this is `ocamlPackages`? Does the behavior of the `opam2json` binary depend on the version of the OCaml compiler used to build it? If not, please let it out of `ocamlPackages`.

Reply:
Like this?

Reply:
Looks better. Thanks!